Code/Framework/AzToolsFramework/AzToolsFramework/UI/Prefab/Constants.h

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+namespace AzToolsFramework
+{
+    namespace PrefabUIConstants
+    {
+        static const int prefabCapsuleRadius = 6;
+        static const int prefabBorderThickness = 2;
+        static const int prefabFileNameFontSize = 10;
+        static const int prefabEditIconSize = 16;
+
+        static const int levelDividerThickness = 1;
```

Comment:
Do we need these to be static if they're in a namespace?

Reply:
No, it's not needed. They are implicitly static!

https://stackoverflow.com/a/177781/5284451